### ISSUE TITLE
Strange Game Timing Issue then crash in Miles Morales with 525

### ISSUE THREAD

**Issue by tfinnegan937** [pop-os/nvidia-graphics-drivers#172]
When playing Spider-Man Miles Morales with Nvidia driver 525 installed, the game runs at full speed, but the animation in the background of the main menu will start and stop randomly (Independent of the actual game speed). After a minute or two of this, the game crashes altogether. This does not occur on 515

**Comment by mmstick** [comment]
Any chance that this was simply shader compilation for a game that's not been run by this driver on Linux yet?

**Comment by tfinnegan937** [comment]
Good catch. I usually skip the precompilation and let them compile in the background. I'll give that a try and report back. On 515, it was not crashing when I did so, however.

**Comment by tfinnegan937** [comment]
I have completely deleted and reinstalled steam and the game. I allowed for shaders to fully precompile. The game still crashes at startup on the latest drivers. I'm not sure if this is on the driver, or the game, though. Is there more information that I can provide to narrow this down?

**Comment by n3m0-22** [comment]
This may be relevant. https://github.com/ValveSoftware/Proton/issues/6328

**Comment by mmstick** [comment]
Could be related to https://www.phoronix.com/news/DXVK-NVAPI-0.6
